src/configs/battery.rs

```diff
@@ -23,17 +24,42 @@ impl<'a> RootModuleConfig<'a> for BatteryConfig<'a> {
             unknown_symbol: "",
             empty_symbol: "",
             format: "[$symbol$percentage]($style) ",
-            display: vec![BatteryDisplayConfig {
-                threshold: 10,
-                style: "red bold",
-            }],
+            display: vec![BatteryDisplayConfig::new()],
             disabled: false,
         }
     }
 }
 
-#[derive(Clone, ModuleConfig)]
+#[derive(Clone)]
 pub struct BatteryDisplayConfig<'a> {
     pub threshold: i64,
     pub style: &'a str,
+    pub symbol: &'a str,
+}
+
+impl<'a> RootModuleConfig<'a> for BatteryDisplayConfig<'a> {
```

Comment:
Just `impl Default` instead here (and please rebase this pr because there's been some changes related to `RootModuleConfig`).

Reply:
Thanks for your comments.
I changed `RootModuleConfig` to `impl Default`, and thus the build was successful.
Also, I did `git rebase`. If there is any mistake I made, please let me know.